src/components/GraphFilter/GraphFind.tsx

```diff
@@ -177,23 +177,24 @@ export class GraphFind extends React.PureComponent<GraphFindProps, GraphFindStat
                 </OverlayTrigger>
               )}
             </InputGroup>
-            <OverlayTrigger
-              key={'ot_graph_find_help'}
-              placement="top"
-              overlay={<Tooltip id={'tt_graph_find_help'}>Find/Hide Help...</Tooltip>}
-            >
-              <Button bsStyle="link" style={{ paddingLeft: '6px' }} onClick={this.toggleFindHelp}>
-                <KialiIcon.Help className={defaultIconStyle} />
-              </Button>
-            </OverlayTrigger>
+            <GraphHelpFind onClose={this.toggleFindHelp}>
+              <OverlayTrigger
```

Comment:
There is no `OverlayTrigger` in PF4. Just use [Popover](https://patternfly-react.surge.sh/patternfly-4/documentation/react/components/popover)